agent/bench-scripts/test-bin/java

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
```

Comment:
Shouldn't we have a nice block comment here explaining what this blob of elegance is for and what it does?

Reply:
Yes, I should write that it is a mock of the expected behavior of `specjbb2005`.